src/pages/authentication/connect.md

```diff
@@ -150,6 +105,45 @@ const authOptions = {
 blockstackConnect.showBlockstackConnect(authOptions);
 ```
 
+### In React Apps
+
+If you're using `connect` in a React app, then the best option is to use the package `@blockstack/connect-react`, utilizing React Provider and hooks in your React app.
```

Comment:
We should explicitly say `[React Context Provider](https://reactjs.org/docs/context.html)`